cpu/stm32/periph/dma.c

```diff
@@ -479,10 +486,30 @@ void dma_resume(dma_t dma, uint16_t remaining)
     int stream_n = dma_config[dma].stream;
     STM32_DMA_Stream_Type *stream = dma_ctx[dma].stream;
 
+#if CPU_FAM_STM32F2 || CPU_FAM_STM32F4 || CPU_FAM_STM32F7
```

Comment:
Can we do
```suggestion
#ifdef DMA_SxCR_MINC
```

Reply:
Yes, I almost did that. I will change it here, and maybe a few other places.